examples/features/interceptor/client/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary client is an example client.
+// Client shows how to use interceptors for logging and fallback
```

Comment:
i think interceptors are not restricted to just logging and fallback. We can probably steal the sentences from readme or grpc.io